docs/document/docs/components/desktop/readonly-table.md

```diff
@@ -20,15 +24,21 @@ Here is a list of properties that can be used for modifying the component:
 
 | Name   | Type | Default | Description | Remark |
 | :--- | :--- | :--- | :--- | :--- |
-| className | string | ""  | Component class name | |
-| id | string | ""  | Component id name | |
-| label | string | ""  | Label for the component | Label will not be displayed if unspecified or left empty |
-| visible | boolean | true | Show/Hide the component | |
-| columns | Array\<Columns\> | []  | Array of data to be displayed in the table header | Will result an error if the value for columns is not an array |
-| Columns.header | Object | Null | Table header object | |
-| Columns.header.text | String | Null | Text to be displayed in the table header | |
-| Columns.visible | Boolean | Null | Show/Hide the column | |
-| data | Array\<Array\<string\>\> | []  | Two-dimensional array of data to be displayed on table body | Will result an error if the data is not an array |
+| className | string | ""  | Component class name. | |
+| id | string | ""  | Component id name. | |
+| label | string | ""  | Text description for the component. | Label will not be displayed if unspecified or left empty. |
+| rowsPerPage | number | 5 | Number of table rows per page. | <ul><li>Round off to the nearest whole number. </li><ul> <li> `{rowsPerPage: 10}` Display up to 10 rows per page. </li><li> `{rowsPerPage: 20.5}` Display up to 21 rows per page. </li> <li> `{rowsPerPage: 20.4}` Display up to 20 rows per page. </li></ul><li>Throw an error if `rowsPerPage` is not a positive number in constructor and setter.</li><ul>|
+| pagination | boolean | true | Show/Hide the pagination. | <ul><li>`{pagination: false}` Pagination is hidden, all rows are displayed.</li><li>`{pagination: true}` Only the number of rows set in rowsPerPage are displayed.</li></ul> |
+| visible | boolean | true | Show/Hide the component. | |
+| columns | Array\<[Columns](#column)\> | []  | Column data of the component. | Throw an error if the `columns` is not an array. |
+| data | Array\<object\> | []  | Row data of the component. | Throw an error if the `data` is not an array. |
+
+#### Column
```

Comment:
How about writing inside "### Property" like this?
Column.field, Column.title, etc.
https://kintone-ui-component.netlify.app/docs/en/components/desktop/dropdown